amplience-automation/automation-files/schema/schemas/static-page-schema.json

```diff
@@ -0,0 +1,53 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "https://sfcc.com/staticpage",
+
+	"title": "Title",
+	"description": "Description",
+
+	"allOf": [
+		{
+			"$ref": "http://bigcontent.io/cms/schema/v1/core#/definitions/content"
+		}
+	],
+	
+	"type": "object",
+	"properties": {
+		"_meta": {
```

Comment:
The indentation here is generally wrong - it switches to 4 space when the rest of the indentation is tabs.

Reply:
updated